file-formats/chko/type/zif_atc_aff_chko_v1.intf.abap

```diff
@@ -0,0 +1,54 @@
+INTERFACE zif_atc_aff_chko_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Parameter</p>
+    "! A parameter of an ATC Check Object
+    BEGIN OF ty_parameter,
+      "! <p class="shorttext">Technical ID</p>
+      "! The technical Id of a parameter
+      technical_id TYPE c LENGTH 4,
```

Comment:
I am not sure, whether this is really needed

Reply:
We use a type which is a hashed table with unique key technical_id. As the persistence class transfers our type into this type, there should be this component.